tensornetwork/contractors/opt_einsum_paths/path_contractors.py

```diff
@@ -62,9 +126,9 @@ def base(net: TensorNetwork,
   if (len(final_node.edges) <= 1) and (output_edge_order is None):
     output_edge_order = list((net.get_all_edges() - net.get_all_nondangling()))
   elif (len(final_node.edges) > 1) and (output_edge_order is None):
```

Comment:
I think you can remove this check now since you raise it earlier?

Reply:
This check actually needs to be there. I changed the check in`_base_nodes` before. So, I also tried to remove the code duplication of `_base_network` and `_base_node`, but it turns out to lead to some annoying typing problems (essentially, pytype seems not to like functions returning different types) which I couldn't easily fix. So I left it as is for now and added a todo.